pkg/odo/cli/catalog/describe/service.go

```diff
@@ -33,6 +39,15 @@ type DescribeServiceOptions struct {
 	plans   []svc.ServicePlan
 	// generic context options common to all commands
 	*genericclioptions.Context
+
+	backend CatalogProviderBackend
+
+	// Operator backed services params
+	// the operator name
+	OperatorType   string
+	CustomResource string
+	CSV            olm.ClusterServiceVersion
+	CR             *olm.CRDDescription
```

Comment:
Do we need these elements here? IMO they should be a part of the `opreatorBackend` struct.

Reply:
Which elements specifically?

Reply:
moved all

Reply:
My bad, I was referring to the four elements that are specific to Operators - `OperatorType`, `CustomResource`, `CSV` and `CR`.